SU-SWS/saa_victoria_subtheme
# ADAPT-000 Update vivid red variables to use SAA's vivid red

# READY FOR REVIEW

# Summary
- Update vivid red variables to use SAA's vivid red

# Review By (Date)
- asap

# Urgency
- high

# Steps to Test

1. Pull this branch to local
2. Clear site cache
3. Review navigation hover and in-page banner. The color should be `#e50808`

# Associated Issues and/or People
- ADAPT-476

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
